<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardServer</class>
 <widget class="QMainWindow" name="CardServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Сервер</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icos.qrc">
    <normaloff>:/1.png</normaloff>:/1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="6" rowspan="3" colspan="2">
     <widget class="QListWidget" name="terminals">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="1" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Логи сервера:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="12">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="7">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="5" rowspan="6">
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="9">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID клиента:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Список клиентов:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2" colspan="5">
     <widget class="QPushButton" name="addCardButton">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Создать новую карту</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="Line" name="line_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="3" colspan="10">
     <widget class="QPushButton" name="clearlog">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Очистить логи сервера</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9" colspan="3">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Добавить клиента:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="11">
     <widget class="QPushButton" name="removeTerminalButton">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="12">
     <widget class="QTableWidget" name="cards">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Номер карты</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Баланс</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Месяц</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Год</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PIN</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Имя и Фамилия</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Украдена</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Утеряна</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Вводов PIN</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="Line" name="line_10">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Список карт:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8" rowspan="6">
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="8" rowspan="2">
     <widget class="Line" name="line_9">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="10" colspan="2">
     <widget class="QLineEdit" name="idEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="12" column="1" colspan="12">
     <widget class="QListWidget" name="log">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item row="2" column="12" rowspan="6">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="3">
     <widget class="QPushButton" name="startServerButton">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Запустить сервер</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Настройки сервера:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" rowspan="2">
     <widget class="Line" name="line_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="7" colspan="6">
     <widget class="QPushButton" name="removeCardButton">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Удалить выбранную карту</string>
      </property>
     </widget>
    </item>
    <item row="0" column="12" rowspan="2">
     <widget class="Line" name="line_11">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="9" colspan="2">
     <widget class="QPushButton" name="addTerminalButton">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Порт сервера:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="portEdit"/>
    </item>
   </layout>
  </widget>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="loadFromFIle">
   <property name="text">
    <string>Загрузить настройки из файла</string>
   </property>
   <property name="toolTip">
    <string>Загрузить настройки из файла</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="saveToFile">
   <property name="text">
    <string>Сохранить настройки в файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icos.qrc"/>
 </resources>
 <connections/>
</ui>
